src/test/mochitest/browser.ini

```diff
@@ -79,6 +80,7 @@ skip-if = os == "win" # Bug 1393121
 [browser_dbg-search-symbols.js]
 skip-if = os == "win" # Bug 1393121
 [browser_dbg-search-project.js]
+skip-if = true # Bug 1393121, 1393299
```

Comment:
we should also get serious about removing these skips. e.g. I'm pretty sure we fixed the intermittent w/ searching, so we can remove the search skips after the 57 release